examples/explorer-template/packages/client/src/App.tsx

```diff
@@ -0,0 +1,29 @@
+import { useComponentValue } from "@latticexyz/react";
+import { useMUD } from "./MUDContext";
+import { singletonEntity } from "@latticexyz/store-sync/recs";
+
+export const App = () => {
+  const {
+    components: { Counter },
+    systemCalls: { increment },
+  } = useMUD();
+
+  const counter = useComponentValue(Counter, singletonEntity);
+
+  return (
+    <>
+      <div>
+        Counter: <span>{counter?.value ?? "??"}</span>
```

Comment:
thoughts on using the plain react template rather than the ECS one? so we get a task list instead of a counter

Reply:
Yeah, task list is a more interactive example so makes sense to use it as well, changed to it